app/views/projects/merge_requests/show/_mr_accept.html.haml

```diff
@@ -1,12 +1,12 @@
 - unless @allowed_to_merge
   - if @project.archived?
     %p
-      %strong Archived projects cannot be committed to!
+      %strong Archived projects do not provide commit access, sorry.
```

Comment:
I think this should be "Archived projects do not provide commit access." without the apology.